apps/kevin-malone/bot/common/threads/onboarding.py

```diff
@@ -73,7 +70,33 @@ async def control_hook(self, message, user_id):
                 user_id, self.cache, USER_DB_ID_CACHE_KEY, user["id"]
             )
             return StepKeys.ASSOCIATE_EXISTING_USER_WITH_GUILD.value
-        return StepKeys.PROMPT_USER_WALLET_ADDRESS.value
+        return StepKeys.PROMPT_USER_CONNECT_WALLET.value
+
+
+class PromptUserToConnectWallet(BaseStep):
+    """Step to submit wallet address for the govrn profile"""
+
+    name = StepKeys.PROMPT_USER_CONNECT_WALLET.value
+
+    def __init__(self, cache, guild_id, bot):
+        super().__init__()
+        self.cache = cache
+        self.guild_id = guild_id
+        self.bot = bot
+
+    async def send(self, message, user_id):
+        channel = message.channel
+        discord_user = await self.bot.fetch_user(user_id)
+        msg = (
+            "It doesn't look like your wallet"
+            " is connected to this discord account please "
```

Comment:
In this case, we don't know if the user already has an account setup with Govrn, but has not yet linked their discord, or if this is their first time actually joining, right? Can we update the copy to the following to reflect instructions for a first time join? i.e.: "Hello! We don't have a Govrn profile associated with your discord quite yet. If you'd like to setup a Govrn profile, or associate your discord with an existing Govrn profile, you can do so here: {constants.Bot.frontend_url}/#/signature?displayName={discord_user.display_name}. After that, just run /join again to hop into your guild!"

Reply:
Doing now